plugin/pkg/scheduler/scheduler.go

```diff
@@ -140,26 +145,26 @@ func NewFromConfigurator(c Configurator, modifiers ...func(c *Config)) (*Schedul
 }
 
 // Run begins watching and scheduling. It starts a goroutine and returns immediately.
-func (s *Scheduler) Run() {
-	go wait.Until(s.scheduleOne, 0, s.config.StopEverything)
+func (sched *Scheduler) Run() {
```

Comment:
which version of golint? I think we used this kind of varname in lots of funcs; what's golint's output for others?